src/bubblesort.ts

```diff
@@ -1,3 +1,13 @@
 export function bubblesort(inputArray) {
-  return inputArray;
+  for (let i=0; i < inputArray.length; i++) {
+    for (let j=0; j<inputArray.length; j++) {
+      if (inputArray[j] > inputArray[j+1]) {
+        let num = inputArray[j];
+        inputArray[j] = inputArray[j+1];
+        inputArray[j+1] = num;
+      }
+    }
+  } 
+  return inputArray; 
 }
+
```

Comment:
You have an outer loop that assigns variable `i` and an inner loop that assigns variable `j`.

`i` is never used. Is that intended? Does this pass the test?

Reply:
It did before I submitted.  Now, it does not.  In fact, nothing appears to be working.

Reply:
I would recommend committing (`git add .` `git commit -m 'message'`) frequently. You can always rewind changes if it's in git. BUt if you don't put it in you'll end up in cases like this.

Even when my code is messy, when I get things working I always git commit with a message like "working prototype" so that I can never lose the changes

Reply:
> You have an outer loop that assigns variable `i` and an inner loop that assigns variable `j`.
> 
> `i` is never used. Is that intended? Does this pass the test?

I don't see how the code will execute without nested loops. Please advise.

Reply:
I'm not actually commenting on the code or implementation. I'm asking you to think about why you have the code setup the way you do. Theres one code smell that stands out: you have an outer loop but *do nothing with it*. You have a variable `i` defined in that outer loop, but never use `i`. This tells me that you either:
* don't need the loop
* forgot to use `i`
* misunderstand how/when to use loops

Reply:
The outer loop allows for the inner loop to make repeated passes.

Reply:
I see what's happening. 

It looks like this does work; though it would be helpful for you to add extra tests to ensure proper correctness.

On top of that, we're still not using the variable in the outer loop. Do you think you can use it to improve performance?